src/renderer/lib/hooks/useQuickStart.ts

```diff
@@ -64,6 +72,7 @@ export const useQuickStart = () => {
     hasIso: Boolean(savedIsoPath),
     hasVerifiedEmail: Boolean(user?.emailVerified),
     hasPlayKey: Boolean(playKey),
+    rulesAccepted: playKey?.rulesAccepted ?? 0,
```

Comment:
I realize the playKey is not being used as intended here. Though generally how I would expect things to work is that there would be an apollo store that would fetch all user info up-front. Not really sure how to achieve this at the moment.

Reply:
i don't think its a big deal. one thing I do wonder is if it would be better to do the boolean check here instead of in the if statement.